specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -51,6 +51,583 @@
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.Communication.RouterJobExceptionTriggered event",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AcsRouterJobEventData"
+        }
+      ],
+      "properties": {
+        "ruleKey": {
+          "description": "Router Job Exception Triggered Rule Key",
+          "type": "string"
+        },
+        "exceptionRuleId": {
+          "description": "Router Job Exception Triggered Rule Id",
+          "type": "string"
+        }
+      }
+    },
+    "AcsRouterJobQueuedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.Communication.RouterJobQueued event",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AcsRouterJobEventData"
+        }
+      ],
+      "properties": {
+        "priority": {
+          "description": "Router Job Priority",
+          "type": "integer",
+          "format": "int32"
+        },
+        "attachedWorkerSelectors": {
+          "description": "Router Job Queued Attached Worker Selector",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AcsRouterWorkerSelector"
+          }
+        },
+        "requestedWorkerSelectors": {
+          "description": "Router Job Queued Requested Worker Selector",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AcsRouterWorkerSelector"
+          }
+        }
+      }
+    },
+    "AcsRouterJobReceivedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.Communication.RouterJobReceived event",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AcsRouterJobEventData"
+        }
+      ],
+      "properties": {
+        "jobStatus": {
+          "description": "Router Job Received Job Status",
+          "type": "string",
+          "enum": [
+            "PendingClassification",
+            "Queued",
+            "Assigned",
+            "Completed",
+            "Closed",
+            "Cancelled",
+            "ClassificationFailed",
+            "Created",
+            "PendingSchedule",
+            "Scheduled",
+            "ScheduleFailed",
+            "WaitingForActivation"
+          ],
+          "x-ms-enum": {
+            "name": "acsRouterJobStatus",
+            "modelAsString": true
+          }
+        },
+        "classificationPolicyId": {
+          "description": "Router Job Classification Policy Id",
+          "type": "string"
+        },
+        "priority": {
+          "description": "Router Job Priority",
+          "type": "integer",
+          "format": "int32"
+        },
+        "requestedWorkerSelectors": {
+          "description": "Router Job Received Requested Worker Selectors",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AcsRouterWorkerSelector"
+          }
+        },
+        "scheduledOn": {
```

Comment:
Also, just to confirm - these events are not being published yet, right?

Reply:
the are live in the portal